src/main/kotlin/apply/domain/applicationform/DuplicateUserValidator.kt

```diff
@@ -5,12 +5,12 @@ import apply.domain.recruitment.getById
 import org.springframework.stereotype.Component
 
 @Component
-class DuplicateApplicationValidator(
+class DuplicateUserValidator(
```

Comment:
`DuplicateApplicationValidator`이 맞습니다.

```suggestion
class DuplicateApplicationValidator(
```

Reply:
Validator는 Application이 맞겠네요... 
마구잡이 바꾸느라 놓친것 같은데 반성합니다 😥